src/desktop/models/page.coffee

```diff
@@ -1,9 +1,17 @@
-_ = require 'underscore'
-sd = require('sharify').data
 Backbone = require 'backbone'
+sd = require('sharify').data
 { Markdown } = require 'artsy-backbone-mixins'
+sanitizeHtml = require('sanitize-html')
+_ = require 'underscore'
 
 module.exports = class Page extends Backbone.Model
+
   _.extend @prototype, Markdown
 
   urlRoot: "#{sd.API_URL}/api/v1/page"
+
+  sanitizedHtml: (attr) ->
+    clean = sanitizeHtml(@get(attr), {
+      allowedTags: sanitizeHtml.defaults.allowedTags.concat([ 'h1', 'h2', 'summary', 'details' ])
```

Comment:
Interesting that the [default allowed tags](https://github.com/apostrophecms/sanitize-html#what-are-the-default-options) include h3 to h6 but not h1 or h2.

Reply:
Oh yeah, I wasn't sure why that was. I chose these because I think they'll be useful for the first round of edits that the auction team wants to make, and it seemed safe!

Reply:
We can probably move the list to a constant or something for easier extension.